android/app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,5 @@
   <string name="exposure_notification_channel_name">Exposure Notification</string>
   <string name="exposure_notification_message">Someone you were near has tested positive for COVID-19. Tap for more info.</string>
   <string name="exposure_notification_title">Possible COVID-19 exposure</string>
+  <string name="background_worker_notification_title">Checking exposures…</string>
```

Comment:
What should we show while we are downloading diagnosis keys?
It also needs to be translated

Reply:
I think this copy is fine.  We should be able to add these to the android translations.  Happy to do it if you wanna cut card for it.

Reply:
Sure!